burotel/indico_burotel/client/js/reducers.js

```diff
@@ -0,0 +1,12 @@
+import _ from 'lodash';
+
+
+export default [(state, action) => {
+    if (action.type === '@@INIT' || action.type === 'RESET_PAGE_STATE') {
```

Comment:
what about importing the constant instead of hardcoding the `RESET_PAGE_STATE` string. also, I think `@@INIT` is devtool-only and should be our custom `'INIT'` instead (we also have a constant for this that could be imported)